homeassistant/components/yalexs_ble/sensor.py

```diff
@@ -15,30 +27,90 @@
 from .models import YaleXSBLEData
 
 
+@dataclass
+class YaleXSBLERequiredKeysMixin:
+    """Mixin for required keys."""
+
+    value_fn: Callable[[LockState, LockInfo, ConnectionInfo], int | float | None]
+
+
+@dataclass
+class YaleXSBLESensorEntityDescription(
+    SensorEntityDescription, YaleXSBLERequiredKeysMixin
+):
+    """Describes Yale Access Bluetooth sensor entity."""
+
+
+SENSORS: tuple[YaleXSBLESensorEntityDescription, ...] = (
+    YaleXSBLESensorEntityDescription(
+        key="",  # No key for the original RSSI sensor unique id
+        name="Signal strength",
+        device_class=SensorDeviceClass.SIGNAL_STRENGTH,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.MEASUREMENT,
+        has_entity_name=True,
+        native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+        entity_registry_enabled_default=False,
+        value_fn=lambda state, info, connection: connection.rssi,
+    ),
+    YaleXSBLESensorEntityDescription(
+        key="battery_level",
+        name="Estimated battery level",
```

Comment:
battery level is always estimated right? It makes the name very long. I would just make it "Battery"

Reply:
Yes of course it is :)

db3a1c4b0eb2692d5d358828a70b568c2babc47f